src/main/java/com/fasterxml/jackson/dataformat/xml/XmlMapper.java

```diff
@@ -280,6 +288,20 @@ public XmlMapper setDefaultUseWrapper(boolean state) {
         return this;
     }
 
+    /**
+     * @since 2.14
+     */
+    public void setXmlTagProcessor(XmlTagProcessor tagProcessor) {
```

Comment:
Let's also remove this mutator; should be passed via `XmlFactory` (and then avoids requiring the other setter in factory)